aws-samples/amazon-chime-sdk-smart-video-sending-demo
# Update webpack to V5 and upgrade dependencies version

**Issue #:**

**Description of changes:**
Upgrade dependencies to address security alarms.
* Upgrade webpack to V5
   * Upgrade the `webpack.config.js`, same as the one in react meeting demo

* Upgrade below dependencies:
   * `concurrently`: from 5.2.0 to 6.4.0
   * `html-webpack-plugin`: from 3.2.0 to 5.5.0
   *  `webpack`: from 4.41.6 to 5.65.0
   *  `webpack-cli`: from 3.3.11 to 4.9.1
   *  `webpack-dev-server`: from 3.11.0 to 4.6.0
* Remove `html-webpack-inline-source-plugin`
* Add `react-dev-utils`

**Testing**

1. How did you test these changes?
   * Check on my forked branch the security issues are addressed.
   * Run `npm run start` and confirm the App works well
   * Deploy the serverless demo and confirm the App works well.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
